manifest.sh

```diff
@@ -109,4 +122,16 @@ COPY_RULES="
 
     src=examples/pipenv-test-app/.gitignore
     dest=test/pipenv-test-app/.gitignore;
+
+    src=examples/micropipenv-test-app/testapp.py
+    dest=test/micropipenv-test-app/testapp.py;
+
+    src=examples/micropipenv-test-app/setup.py
+    dest=test/micropipenv-test-app/setup.py;
+
+    src=examples/micropipenv-test-app/.s2i/environment
+    dest=test/micropipenv-test-app/.s2i/environment;
+
+    src=examples/micropipenv-test-app/.gitignore
+    dest=test/micropipenv-test-app/.gitignore;
 "
```

Comment:
Does `manifest.sh` contain all the new files? I might be looking at it wrong, but for example I don't see the new file `examples/micropipenv-test-app/.s2i/environment`, and there's probably others.

Reply:
The file you mentioned `examples/micropipenv-test-app/.s2i/environment` is in the code snippet you commented on line 133 :confused: 

Nonetheless, missing files or some other errors in manifest would cause problems with distgen or non-working tests. Don't forget that some files are copied, some are symlinks and some are generated via distgen, so they appear in different parts of manifest.